.github/workflows/main.yml

```diff
@@ -0,0 +1,27 @@
+name: Main
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: "30 2 * * *"
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.6
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.6
```

Comment:
3.6? A není to málo, Antone Pavloviči?

Reply:
Tenhle PR přepíná na GH Actions, nemění ještě k tomu verzi Pythonu.
(Samozřejmě by to časem stálo za to změnit taky, ale je lepší mít najednou míň věcí co se mužou rozbít. A 3.6 je stále podporovaná)